src/js/components/general/banner.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classnames from 'classnames';
+
+const Banner = ({ currentLocation }) => {
+
+    let classes = classnames("banner", {
+        // "display-none": true
```

Comment:
do we need classnames if the display none is commented out? 

just add banner to the className in the div below.